setup.py

```diff
@@ -1,6 +1,57 @@
+import io
+import re
 from setuptools import setup, find_packages
-import sys
 
+
+def read_file(filename, **kwargs):
+    encoding = kwargs.get("encoding", "utf-8")
+
+    with io.open(filename, encoding=encoding) as f:
+        return f.read()
+
+
+def replace_local_hyperlinks(
+        text, base_url="https://github.com/project-rig/rig/blob/master/"):
+    """Replace local hyperlinks in RST with absolute addresses using the given
+    base URL.
```

Comment:
"This is used to make links in the long description function correctly outside of the repository (e.g. when published on PyPi)."